pkgs/by-name/al/almo/package.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  fetchFromGitHub,
+  stdenv,
+  gcc,
+  python312Packages,
+}:
+let
+  version = "v0.9.5-alpha";
+in
+stdenv.mkDerivation {
+  pname = "almo";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "abap34";
+    repo = "almo";
+    rev = "refs/tags/${version}";
+    sha256 = "sha256-Cz+XDJmdp+utzwm1c7ThTNS6kfNF6r4B16tnGQSCVMc=";
+  };
+
+  buildInputs = [
+    gcc
+    python312Packages.pybind11
+  ];
+
+  buildPhase = ''
+    make all
+  '';
+
+  installPhase = ''
```

Comment:
please add `runHook preInstall` and `runHook postInstall`

Reply:
I added it.